src/main/java/org/wso2/siddhi/plugins/idea/completion/SiddhiKeywordsCompletionContributor.java

```diff
@@ -0,0 +1,245 @@
+import org.wso2.siddhi.plugins.idea.completion.executionelements.partition.PartitionCompletionContributor;
+import org.wso2.siddhi.plugins.idea.completion.executionelements.query.QueryCompletionContributor;
+import org.wso2.siddhi.plugins.idea.psi.AnnotationNode;
+import org.wso2.siddhi.plugins.idea.psi.AppAnnotationNode;
+import org.wso2.siddhi.plugins.idea.psi.AttributeNameNode;
+import org.wso2.siddhi.plugins.idea.psi.AttributeTypeNode;
+import org.wso2.siddhi.plugins.idea.psi.DefinitionElementNode;
+import org.wso2.siddhi.plugins.idea.psi.ExecutionElementNode;
+import org.wso2.siddhi.plugins.idea.psi.FunctionDefinitionNode;
+import org.wso2.siddhi.plugins.idea.psi.FunctionNameNode;
+import org.wso2.siddhi.plugins.idea.psi.LanguageNameNode;
+import org.wso2.siddhi.plugins.idea.psi.OutputEventTypeNode;
+import org.wso2.siddhi.plugins.idea.psi.ParseNode;
+import org.wso2.siddhi.plugins.idea.psi.PartitionNode;
+import org.wso2.siddhi.plugins.idea.psi.QueryNode;
+import org.wso2.siddhi.plugins.idea.psi.SiddhiAppNode;
+import org.wso2.siddhi.plugins.idea.psi.SiddhiFile;
+import org.wso2.siddhi.plugins.idea.psi.TriggerNameNode;
+import org.wso2.siddhi.plugins.idea.psi.WindowDefinitionNode;
+
+import javax.annotation.Nonnull;
+
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addAfterATSymbolLookups;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addAtKeyword;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addDefineTypesAsLookups;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addEveryKeyword;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addFromKeyword;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addInitialTypesAsLookups;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addLanguageTypesKeywords;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addOutputEventTypeKeywords;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addReturnKeyword;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addValueTypesAsLookups;
+import static org.wso2.siddhi.plugins.idea.completion.SiddhiCompletionUtils.addWindowProcessorTypesAsLookups;
+import static org.wso2.siddhi.plugins.idea.completion.executionelements.partition.PartitionCompletionContributor.isEndOfAQueryOutput;
+import static org.wso2.siddhi.plugins.idea.completion.util.KeywordCompletionUtils.getPreviousVisibleSiblingSkippingComments;
+
+/**
+ * Defines utility methods used for code completions.
+ */
+public class SiddhiKeywordsCompletionContributor extends CompletionContributor {
+
+    @Override
+    public void fillCompletionVariants(@Nonnull CompletionParameters parameters, @Nonnull CompletionResultSet result) {
+        PsiElement element = parameters.getPosition();
+        if (element instanceof LeafPsiElement) {
```

Comment:
Invert the logic to return from the method sooner.

Reply:
Fixed in commit 3aa87a6